test/e2e/common.go

```diff
@@ -175,7 +175,7 @@ func EnsureImage(k8sVersion string) (imageURL string, imageChecksum string) {
 // DownloadFile will download a url and store it in local filepath.
 func DownloadFile(filePath string, url string) error {
 	// Get the data
-	resp, err := http.Get(url) //nolint:noctx
+	resp, err := http.Get(url) //nolint:noctx //#nosec
```

Comment:
can we test
```suggestion
	resp, err := http.Get(url) //nolint:noctx,gosec 
```
with //#nosec we can also say if we want to skip specific rules only i.e.  `#nosec G401`

Reply:
Sometimes it wants `//#nosec` and sometimes `//nolint:gosec`... anyways, will be fixed. Thanks.